### PR TITLE
[M]Allow longer length of paths fields in VerificationRequest

### PR DESCRIPTION
**Pull request by Jakub89**
Resolves: #598

### CODE REVIEW

**Comment by Jakub89** [comment]
@kbeker I did, there was no problem as expected with such simple change.

File: concent_api/conductor/models.py [hunk 1]
```diff
@@ -19,10 +19,10 @@ class VerificationRequest(Model):
     subtask_id = CharField(max_length=MESSAGE_TASK_ID_MAX_LENGTH)
 
     # Relative path of the .zip file that contains Blender source files for the render.
-    source_package_path = CharField(max_length=255, unique=True)
+    source_package_path = CharField(max_length=1000, unique=True)
```
Comment (dybi) [inline concent_api/conductor/models.py]: Extract it to a constant

File: concent_api/conductor/models.py [hunk 2]
```diff
@@ -19,10 +19,10 @@ class VerificationRequest(Model):
...
 
     # Relative path of the .zip file that contains the rendering result received from the provider.
-    result_package_path = CharField(max_length=255, unique=True)
+    result_package_path = CharField(max_length=1000, unique=True)
```
Comment (dybi) [inline concent_api/conductor/models.py]: same here

File: concent_api/conductor/models.py (hunk 1 above)
Comment (Jakub89) [reply concent_api/conductor/models.py]: Done. I've used `MESSAGE_PATH_LENGTH`.

File: concent_api/conductor/models.py (hunk 2 above)
Comment (Jakub89) [reply concent_api/conductor/models.py]: Done.

**Pull request merged**